pkg/internalapis/featurestates/featurestates.go

```diff
@@ -407,19 +407,23 @@ func getFeatureStates(ctx context.Context) ([]featurestatesv1alpha1.FeatureState
 	//  Retrieve SV FeatureStates configmap
 	fssConfigMap, err := k8sClient.CoreV1().ConfigMaps(supervisorFeatureStateConfigMapNamespace).Get(ctx, supervisorFeatureStatConfigMapName, metav1.GetOptions{})
 	if err != nil {
+		if apierrors.IsNotFound(err) {
+			log.Errorf("supervisor feature switch state with name: %q not found in the namespace: %q", supervisorFeatureStatConfigMapName, supervisorFeatureStateConfigMapNamespace)
+			os.Exit(1)
+		}
 		log.Errorf("failed to retrieve SV feature switch state from namespace:%q with name: %q", supervisorFeatureStateConfigMapNamespace, supervisorFeatureStatConfigMapName)
 		return nil, err
 	}
 
-	log.Infof("successfully retrieved SV feature switch state from namespace:%q with name: %q", supervisorFeatureStateConfigMapNamespace, supervisorFeatureStatConfigMapName)
+	log.Infof("Successfully retrieved SV feature switch state from namespace:%q with name: %q", supervisorFeatureStateConfigMapNamespace, supervisorFeatureStatConfigMapName)
 	var featureStates []featurestatesv1alpha1.FeatureState
 	for feature, state := range fssConfigMap.Data {
 		var featureState featurestatesv1alpha1.FeatureState
 		featureState.Name = feature
 		featureState.Enabled, err = strconv.ParseBool(state)
 		if err != nil {
 			log.Errorf("failed to parse feature state value: %v for feature: %q. Defaulting to false", state, feature)
```

Comment:
Can you remove "Defaulting to false" in the error log?

Reply:
fixed.